src/elements/content-sidebar/activity-feed/task/messages.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @flow
+ * @file i18n messages
+ * @author Box
+ */
+
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    rejectedAssignment: {
+        id: 'be.rejectedAssignment',
```

Comment:
I would scope these also like other ones. Not sure whats the best, but perhaps we can have a convention based on folder:

be.contentSidebar.activityFeed.task.*

(be|boxui).(element|feature|component)....

And then common ones can be just be.something